frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePicker.unit.spec.tsx

```diff
@@ -265,6 +265,25 @@ describe("DatePicker", () => {
             "2020-05-21",
           ]);
         });
+
+        it(`can add time to a specific ${description} date filter`, async () => {
```

Comment:
:x: All new tests pass without the fix

Reply:
Super great catch. Fixed. https://metaboat.slack.com/archives/C505ZNNH4/p1729871695712579